libfastjson.yaml

```diff
@@ -0,0 +1,65 @@
+package:
+  name: libfastjson
+  version: 1.2304.0
+  epoch: 0
+  description: a fast json library for C
+  copyright:
+    - license: MIT
+
+environment:
+  contents:
+    packages:
+      - autoconf
+      - automake
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - libtool
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: 3a8402c1de7c7747c95229db26d8d32fb85a7a52
+      repository: https://github.com/rsyslog/libfastjson
+      tag: v${{package.version}}
+
+  - runs: |
+      libtoolize --copy
+      aclocal -I m4 --install
+      autoheader
+      autoconf
+      automake --foreign --add-missing --force-missing --copy
+      ./configure --prefix=/usr
+      make -j $(nproc)
+      make DESTDIR="${{targets.destdir}}" install
+      make check
```

Comment:
Any validation we do should ideally happen in our `test` pipeline

Reply:
Hmm, I agree that run-time checks should happen in the "test" pipeline.  But this is a build-time check.

The source code and Makefile have to be available to run this set of tests supplied by upstream.  We have this in the build pipeline, but this source code isn't available to us in the run time check.  Respectfully, I disagree and recommend keeping the "make check" tests here.

Reply:
Sounds good to me :)

Reply:
I've updated to use the auto pipelines, but I've left the upstream `make check` as part of the build.  I'm happy for the build to fail, if upstreams build-time tests are not able to pass cleanly.